python/ray/_private/utils.py

```diff
@@ -2076,3 +2080,195 @@ def try_update_ld_library_path(
+                base_path,
+                version_dir_name,
+                "lib",
+                "python" + python_version,
+                "site-packages",
+            )
+    return None, None
+
+
+def set_java_jar_dirs_to_env_vars(
+    java_jar_dirs_list: List[str], context: "RuntimeEnvContext"
+):
+    """Insert the path in RAY_JAVA_JARS_DIRS in the runtime env."""
+    java_jar_dirs = ":".join(java_jar_dirs_list)
+    logger.info(f"Setting java jar dirs {java_jar_dirs} to context {context}.")
+    context.env_vars[runtime_env_constants.RAY_JAVA_JARS_DIRS] = java_jar_dirs
+
+
+def try_parse_default_mount_points(mount_dict: Dict[str, str]):
+    default_mount_path = runtime_env_constants.RAY_PODMAN_DEFAULT_MOUNT_POINTS
+    if default_mount_path:
+        default_mount_path_parts = default_mount_path.split(";")
+        for default_mount_path_part in default_mount_path_parts:
+            if ":" in default_mount_path_part:
+                mount_target_path = default_mount_path_part.split(":")
+                mount_dict[mount_target_path[1]] = mount_target_path[0]
+            else:
+                mount_dict[default_mount_path_part] = default_mount_path_part
+    return mount_dict
+
+
+def try_parse_container_run_options(
+    run_options: List,
+    container_command: List,
+    container_to_host_mount_dict: Dict[str, str],
+):
+    if not run_options:
+        return
+    index = 0
+    while index < len(run_options):
+        run_option = run_options[index]
+        if run_option == "-v":
+            if index == len(run_options) - 1:
+                raise RuntimeError(
+                    "Incorrect mount path command, "
+                    "please check the container field "
+                    "in the run_options field mount path, "
+                    "`-v host_path:container_path`."
+                )
+            next_option = run_options[index + 1]
+            logger.info(f"show the next_option {next_option}")
+            paths = next_option.split(":")
+            if len(paths) != 2:
+                raise RuntimeError(
+                    "Incorrect mount path command, "
+                    "please check the container field "
+                    "in the run_options field mount path, "
+                    f"got {next_option}"
+                )
+            source_path = paths[0].strip()
+            target_path = paths[1].strip()
+            # Iterate over the key of
+            # 'container_to_host_mount_dict'
+            # to find if target_path already exists
+            container_to_host_mount_dict[target_path] = source_path
+            index += 2
+        else:
+            container_command.append(run_option)
+            index += 1
+    return container_command, container_to_host_mount_dict
+
+
+def is_py_executable_startswith_pyenv(py_executable: str):
+    """
+    Check if the python executable starts with pyenv.
+    """
+    if py_executable.startswith(get_pyenv_path()):
```

Comment:
I think it does not need to add function `is_py_executable_startswith_pyenv` and just use py_executable.startswith(get_pyenv_path()) to instead of, because it is quit simple.